src/components/Audio.js

```diff
@@ -1,6 +1,13 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 
 export class Audio extends React.Component {
+  static propTypes = {
+    handleClose: PropTypes.func,
```

Comment:
@vishalnarkhede please include docs and describe func signature

Reply:
Not required anymore!!